news-bundle/src/Resources/contao/dca/tl_news_archive.php

```diff
@@ -300,72 +296,6 @@ public function checkPermission()
 		}
 
 		$GLOBALS['TL_DCA']['tl_news_archive']['list']['sorting']['root'] = $root;
-		$security = System::getContainer()->get('security.helper');
-
-		// Check permissions to add archives
-		if (!$security->isGranted(ContaoNewsPermissions::USER_CAN_CREATE_ARCHIVES))
-		{
-			$GLOBALS['TL_DCA']['tl_news_archive']['config']['closed'] = true;
-			$GLOBALS['TL_DCA']['tl_news_archive']['config']['notCreatable'] = true;
-			$GLOBALS['TL_DCA']['tl_news_archive']['config']['notCopyable'] = true;
```

Comment:
Being `closed`, `notCreatable` or `notCopyable` is synonymous with the respective voters (`DC_ACTION_CREATE`/`DC_ACTION_COPY`), so it is unnecessary to have both.

Reply:
additionally, voters can be overridden using our PriorityStrategy, which these flags cannot

Reply:
We have to check all occurences of `'closed'` in DC_Table and DataContainer then I think. 
For example the buttons “save and new” and “save and duplicate” are shown now even if the user is not allowed to create an archive.

Not sure if this is in the scope of this PR though.

Reply:
Refactoring buttons and global operations is a follow-up task after https://github.com/contao/contao/pull/4663. But this PR is the foundation to actually see the permissions being applied.

Reply:
We'll have to apply permissions on the buttons as well, we know that. However, configuring the whole DC dynamically to be `notCopyable` or `notCreatable` based on permissions has always been an artchitectural design issue. These are settings and should be immutable by definition. We can bring back the current behaviour by checking e.g. the `copy` permissions for all the IDs listed in order to restore the current behavior. It would also allow for future UX improvements like e.g. adding `data-` attributes to every single entry and using JS in order to give the users hints about e.g. why there's no copy button or which ones of them can be copied but not deleted etc.

Reply:
Totally agree 👍
Removing this here is correct IMO, and permissions for “save and new”, “save and duplicate” and the like is for another PR.